bool bola_state
string bola_inFrame_Pos

